lib/modules/modules.go

```diff
@@ -138,7 +148,10 @@ type AccessMonitoringFeature struct {
 
 // ToProto converts Features into proto.Features
 func (f Features) ToProto() *proto.Features {
+	productTypeValue := proto.ProductType_name[int32(f.ProductType)]
```

Comment:
This seems like an identity function (we convert e.g. `1` to `"PRODUCT_TYPE_TEAM"`, ten back to `1`). Can't we just do `proto.ProductType(f.ProductType)`, as long as the values in both "enums" match?